src/oracle/connection/stmt.rs

```diff
@@ -115,7 +115,7 @@ impl Statement {
             sizes: Vec::with_capacity(NUM_ELEMENTS),
             indicators: Vec::with_capacity(NUM_ELEMENTS),
             mysql,
-            returning_buffer: Vec::with_capacity(NUM_ELEMENTS),
+            returning_contexts: Vec::new(),
```

Comment:
Why not `with_capacity?`

Reply:
what's the point? We reserve it below with `reserve_exact` anyway.

Reply:
Curiosity.